eZ/Publish/Core/FieldType/RichText/Resources/schemas/ezxhtml5/output/ezxhtml5.xsd

```diff
@@ -249,13 +249,21 @@
     <xs:attributeGroup ref="globalAttributeGroup"/>
   </xs:complexType>
 
+  <xs:complexType name="prePhrasingContentElement">
```

Comment:
AFAIK, it is not possible to add generic support for data-* attributes i xsd schemas. ( no such wildcard functionality exists)

Reply:
The point with `data-language` is a request from @andrerom.
In legacy, you may add a custom tag named `language` to the `<literal>` tag. The idea is to store the value of such a custom attribute (if it is defined) in `data-language`.

But it then might be applicable to add a template rendering step for `<programlisting>`  so user may deal with  that attribute in a template?